source/administration/object-management/data-compression.rst

```diff
@@ -0,0 +1,191 @@
+.. _minio-data-compression:
+
+================
+Data Compression
+================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+MinIO Server supports compressing objects to reduce disk usage.
+Objects are compressed before they are written to disk and uncompressed when read, making the compresson process transparent to client applications and services.
+
+Depending on the type of data, compression may also increase overall throughput.
+Write throughput for compressible data is generally 500MB per second or greater per available CPU.
+Decompression is typically at least 1 GB per second.
+
+Data compression is a global option, configured settings apply to all buckets in a deployment.
+You can control which objects are compressed by specifying the desired file extensions and media types.
+
+.. admonition:: Existing objects are not modified
+   :class: note
+
+   Enabling, disabling, or updating a deployment's compression settings does not modify existing objects.
+   New objects are compressed according to the settings in effect at the time they are created.
+
+
+Incompressible Data
+~~~~~~~~~~~~~~~~~~~
+
+Some types of objects cannot be effectively compressed, such as video or already compressed data.
+MinIO does not compress common incompressible file types, even if they are specified in the compression configuration.
+
+Objects of these types are never compressed:
+
+File extensions
+
+- ``gz``, ``bz2``, ``rar``, ``zip``, ``7z``, ``xz``, ``mp4``, ``mkv``, ``mov``
+
+Media types
+- ``video/*``, ``audio/*``, ``application/zip``, ``application/x-gzip``, ``application/x-bz2``, ``application/x-compress``, ``application/x-xz``
```

Comment:
TODO: some reasonable table format to show these. (Also in several other locations, may become an include?)

Reply:
Could do an unordered list + the `.. dropdown::` to let users manually expand each list to view the contents.